scripts/getValidators_docker

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# find node folder from docker inspect result
+node_folder=$(docker inspect 0-Node | sed -n 's/^.*"Source"\: "\(.*\)\/[0-9]-Node.*$/\1/p')
```

Comment:
Can we get this from an environment variable instead?
This is probably more accurate but, just thinking to simplify it. Not really and issue.

Reply:
yeah, agree, this will be much simpler